src/app/views/infoblock/infoblock.html

```diff
@@ -0,0 +1,4 @@
+<div class="panel-content text-center">
+  <h2 class="{{ $ctrl.colorClass }}">{{ $ctrl.value }}</h2>
+  <p class="font-alt color-light text-uppercase">{{ $ctrl.title}}</p>
```

Comment:
That's good to know, I would have assumed that one-time binding wouldn't work since the variable gets set after the element is instantiated. I'll put that in.